.github/workflows/image-push-master.yml

```diff
@@ -5,6 +5,7 @@ on:
       - master
 jobs:
   build-and-push-amd64-device-plugin:
+    if: github.repository == 'k8snetworkplumbingwg/sriov-network-device-plugin'
```

Comment:
nit: any chance to have this reference a variable ?

similar to[1]

[1] https://github.com/k8snetworkplumbingwg/multus-cni/blob/89188023ef7d07b9a5a77633259a870f772d54ed/.github/workflows/image-push-master.yml#L6

Reply:
done